components/layout/layout_task.rs

```diff
@@ -135,6 +135,9 @@ pub struct LayoutTask {
     /// The URL of the pipeline that we belong to.
     pub url: Url,
 
+    /// Is the current reflow of an iframe, as opposed to a root window?
+    pub iframe: bool,
```

Comment:
nit: `is_iframe` might be better (throughout this PR). A parameter of "iframe" sounds like it contains a reference to a frame object